### PR TITLE
Alasql data type improvement

### PR DESCRIPTION
**Pull request by drelliche**
This pr changes two things about the data in the alasql tables:

1. numeric data is now numeric in the following columns:
 - lat, lon, expenses, coverage in the patients table
 - observation_value in observations table

2. Null values are now `undefined` instead of the string 'NA'. This means that they no longer appear as NA in the tables, now those entries are blank, but they do actually serve as null values for AVG, MAX, MIN, etc.

Heads up that the second change may impact some of the other SQL modules... 

Also, note that zip can't be numeric because they all start with 0 up there in Massachusetts lol

### CODE REVIEW

**Comment by drelliche** [comment]
@pm0kjp Could you take a look at the SQL intermediate changes? Once you do, we will have to change the macros to point back to main.

**Comment by pm0kjp** [comment]
https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/alasql_data_type_improvement/sql_intermediate/sql_intermediate.md#1 is the link

**Comment by pm0kjp** [comment]
I made a couple of small suggestions around the quiz.  I struggled to understand the second question in the CASE/LIKE quiz so I did reword that, but I'm flexible!

**Review by pm0kjp — CHANGES_REQUESTED** [review]

File: sql_intermediate/sql_intermediate.md [hunk 1]
```diff
@@ -365,26 +319,28 @@ FROM alasql.patients;
 
 *********
 
-You'd like to research patients born in the 1970s (so any year starting 197\_ would work).  Use a `LIKE` statement to enrich the query below and find the patient set you care about.  Below the code box, there's a question.  Stuck?  No worries -- scroll ahead to where the question appears, and if you click the "check mark" button <i aria-hidden="true" class="icon icon-resolve lia-btn__icon"></i> after the question, you'll see the code that we used to answer the question.
+You'd like to research patients born in Massachusetts and care about how far they currently live from their birthplace.  Use a `LIKE` statement to enrich the query below and find the patient set you care about.  Below the code box, there's a question.  Stuck?  No worries -- scroll ahead to where the question appears, and if you click the "check mark" button <i aria-hidden="true" class="icon icon-resolve lia-btn__icon"></i> after the question, you'll see the code that we used to answer the question.
```
[the remark this answers is not in the document]
Comment (drelliche) [reply sql_intermediate/sql_intermediate.md:322]: I like this phrasing better. I had some trouble coming up with a question that could reasonably be asked about this data for the quiz, this is much better phrased!

File: sql_intermediate/sql_intermediate.md [hunk 2]
```diff
@@ -65,7 +66,7 @@ previous_sequential_module: sql_basics
 @end
 
 import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros.md
-import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros_sql.md 
+import: https://raw.githubusercontent.com/arcus/education_modules/alasql_data_type_improvement/_module_templates/macros_sql.md
```
Comment (drelliche) [inline sql_intermediate/sql_intermediate.md:69]:
```suggestion
import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros_sql.md
```

**Comment by drelliche** [comment]
@pm0kjp These were all great suggestions, and I incorporated all of them. I have the change to point the macro back to main as a suggestion, and then I think it is ready to go (until its next revamp)

**Comment by pm0kjp** [comment]
@drelliche I committed the change to point back to main.

**Review by pm0kjp — APPROVED** [review]

**Pull request merged**